tmt/steps/provision/__init__.py

```diff
@@ -1254,45 +1258,124 @@ def show(
 
 @container
 class GuestLog(abc.ABC):
-    # Log file name
+    """
+    Represents a single guest log.
```

Comment:
Just a confusion that I had when reading. The location of the logs is ambiguous at first glance, i.e. is it a file on the guest or on the runner. The naming "guest log" makes that ambigous. Maybe there is more specific wording like "log about a guest", "log owned by a guest"?

Reply:
We don't have a better name, "guest log" is used throughout TF for years (often in the sentence "guest log is empty, why?!" but that a different point...)

> The location of the logs is ambiguous at first glance, i.e. is it a file on the guest or on the runner.

Added more words into the docstring in dd5c1ad2, maybe it's now good enough.